kofta/public/locales/nl/translation.json

```
@@ -0,0 +1,222 @@
+			"soundSettings": "naar geluids-instellingen",
+			"deleteAccount": "account verwijderen"
+		},
+		"notFound": {
+			"whoopsError": "Oops! Deze pagina is kwijtgeraakt in een conversatie.",
+			"goHomeMessage": "Geen zorgen. Jij kan het",
+			"goHomeLinkText": "naar thuispagina"
+		},
+		"room": {
+			"speakers": "Luidsprekers",
+			"requestingToSpeak": "Aanvragen om te mogen spreken",
+			"listeners": "Luisteraars"
+		},
+		"searchUser": {
+			"search": "zoeken..."
+		},
+		"soundEffectSettings": {
+			"header": "Geluiden"
+		},
+		"viewUser": {
+			"editProfile": "profiel bewerken",
+			"followsYou": "volgt jou",
+			"followers": "volgers",
+			"following": "volgt"
+		},
+		"voiceSettings": {
+			"header": "Stem instellingen",
+			"mic": "microfoon:",
+			"permissionError": "geen microfoon gevonden, je hebt hem niet aangesloten of de website heeft geen permissie om je te horen.",
+			"refresh": "microfoon lijst verversen",
+			"volume": "volume:"
+		}
+	},
+	"components": {
+		"avatar": {},
+		"backBar": {},
+		"blockedFromRoomUsers": {
+			"header": "Verbannen gebruikers",
+			"unban": "Verbanning ongedaan maken",
+			"noBans": "niemand is nog verbannen"
+		},
+		"bottomVoiceControl": {
+			"leaveCurrentRoomBtn": "Huidige kamer verlaten",
+			"confirmLeaveRoom": "Weet je het zeker dat je deze kamer wilt verlaten?",
+			"leave": "Verlaten",
+			"inviteUsersToRoomBtn": "Gebruikers uitnodigen",
+			"invite": "Uitnodigen",
+			"toggleMuteMicBtn": "Microfoon dempen schakelen",
+			"mute": "Demp",
+			"unmute": "Dempen opheffen",
+			"makeRoomPublicBtn": "Kamer publiek maken!",
+			"settings": "Instellingen",
+			"speaker": "Luidspreker",
+			"listener": "Luisteraar"
+		},
+		"deviceNotSupported": {
+			"notSupported": "Je toestel is momenteel niet ondersteund. Maak een",
+			"linkText": "kwestie aan op GitHub",
+			"addSupport": "wij zullen proberen uw toestel te laten ondersteunen."
+		},
+		"inviteButton": {
+			"invited": "uitgenodigd",
+			"inviteToRoom": "uitnodigen naar kamer"
+		},
+		"micPermissionBanner": {
+			"permissionDenied": "Toestemming geweigerd om toegang te krijgen tot je microfoon (mogelijk moet je naar de browserinstellingen gaan en de pagina opnieuw laden)",
+			"dismiss": "Sluiten",
+			"tryAgain": "probeer het opnieuw"
+		},
+		"keyboardShortcuts": {
+			"setKeybind": "zet toetsbinding",
+			"listening": "aan het luisteren",
+			"toggleMuteKeybind": "Microfoon dempen schakelen toetsbinding",
+			"togglePushToTalkKeybind": "druk-om-te-praten toetsbinding"
+		},
+		"userVolumeSlider": {
+			"noAudioMessage": "geen mogelijkheid om geluid te ontvangen door een onbekende reden"
+		},
+		"addToCalendar": {
+			"add": "Toevoegen aan kalender"
+		},
+		"wsKilled": {
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
+			"reconnect": "opnieuw verbinding maken"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "publiek",
```

```suggestion
				"public": "openbaar",
```
publiekelijk zou ook een goede vertaling zijn.